app/_how-tos/kic-observability-prometheus-grafana.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Prometheus & Grafana
+description: "Monitor {{ site.base_gateway }} Prometheus metrics using {{ site.kic_product_name}} and Grafana"
+content_type: how_to
+related_resources:
+  - text: KIC Prometheus metrics reference
+    url: /kubernetes-ingress-controller/observability/prometheus/
+
+permalink: /kubernetes-ingress-controller/observability/prometheus-grafana/
+breadcrumbs:
+  - /kubernetes-ingress-controller/
+  - index: kubernetes-ingress-controller
+    section: Observability
+
+products:
+  - kic
+
+tools:
+  - kic
+
+automated_tests: false
+
+works_on:
+  - on-prem
+  - konnect
+
+entities: []
+
+tldr:
+  q: How do I monitor {{ site.base_gateway }} Prometheus metrics using Grafana?
+  a: Deploy a `servicemonitor` Kubernetes resource using the {{ site.base_gateway }} Helm chart, then use a `KongClusterPlugin` to configure the `prometheus` plugin for all services in the cluster.
+
+prereqs:
+  kubernetes:
+    gateway_api: true
+    prometheus: true
```

Comment:
Because there are no entities here, like echo-service, one of the cleanup steps is being rendered incorrectly. Tested this locally, and once I add the echo-service entity here, it renders correctly. I'm not sure how to fix this.
<img width="1006" alt="Screenshot 2025-04-23 at 4 53 48 PM" src="https://github.com/user-attachments/assets/1fc210fe-356b-4bf2-ab93-8460f16f1aa5" />

Reply:
This is an existing issue. https://github.com/Kong/developer.konghq.com/issues/1247 for tracking